src/client/RESTClient.ts

```diff
@@ -39,6 +40,7 @@ export class RESTClient extends EventEmitter {
   readonly product: ProductAPI;
   readonly profile: ProfileAPI;
   readonly user: UserAPI;
+  readonly currencies: CurrencyAPI;
```

Comment:
```suggestion
  readonly currency: CurrencyAPI;
```

Let's go with the "singular" form to be consistent.

Reply:
My bad. Fixed!